README.md

```diff
@@ -157,6 +157,49 @@ query := query.NewQuery(filter.And(
   filter.Not(filter.Contains(Position))))
 ```
 
+If you need to determine if an entity has a component, there is `entry.HasComponent`
+
+For example:
+
+```go
+// We have a query for all entities that have Position and Size, but also any of Sprite, Text or Shape.
```

Comment:
@yohamta Is this correct? or should it be constructed or worded differently?

Reply:
I think it's correct 👍